CONTRIBUTING.md

```diff
@@ -13,7 +13,8 @@ Please note that this project is released with a [Contributor Code of Conduct][c
 
 0. [Fork][fork] and clone the repository
 0. Configure and install the dependencies: `script/bootstrap`
-0. Make sure the tests pass on your machine: `rake`
+0. Setup test fixtures: `rake setup`
+0. Make sure the tests pass on your machine: `rake test`
```

Comment:
To be consistent with ["scripts to rule them all" expectations](https://github.com/github/scripts-to-rule-them-all), I'd suggest creating a quick `script/test` file, even if all it does it run `bundle exec rake test`.